servers/src/main/java/tachyon/worker/Worker.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.worker;
+
+import java.io.IOException;
+
+/**
+ * Interface of a Tachyon worker.
+ */
+public interface Worker {
```

Comment:
there should be another interface `Server` added, and `Worker` and `Master` interface both extend. but that belongs to another PR maybe.